### PR TITLE
[Docs] Added doc on how to handle a large response when using FlyteRemote.sync

### PR DESCRIPTION
**Pull request by Ln11211**
# TL;DR
Updated ` control_plane.rst` doc
## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Added about large response while using Remote.Sync and how to handle it in the ` control_plane.rst` doc. This PR is in reference to https://github.com/flyteorg/flyte/pull/2932#issuecomment-1263393776 and issue [#2809](https://github.com/flyteorg/flyte/issues/2809).

## Tracking Issue
https://github.com/flyteorg/flyte/issues/2809

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by Ln11211** [comment]
@samhita-alla Hey I mentioned about the large response just like you asked. Check from [Line 302](https://github.com/flyteorg/flytekit/pull/1212/files#diff-948e697ac874d98161ff4c6900e3531dcfd26932b27baa2b76f1e777a9d0e0efR302) 

Do you want any changes? :)

**Comment by samhita-alla** [comment]
@Ln11211, I see multiple files that have already been merged into the master branch. Please ensure only the file of concern is present by rebasing the branch.

**Comment by Ln11211** [comment]
> @Ln11211, I see multiple files that have already been merged into the master branch. Please ensure only the file of concern is present by rebasing the branch.

I don't understand what happened, I synced my fork and then used the `git rebase HEAD~5 --signoff` and `git push --force-with-lease origin <your-branch>` to sign off my commits. I guess I messed up the second git command.. How do I rebase the branch?

**Comment by Ln11211** [comment]
Here is what I think I'm supposed to do--
I'll discard my changes(previous commits) and then sync the fork and then commit the changes again, this will fix the issue right?

**Comment by samhita-alla** [comment]
Hopefully! If that doesn't work, please create a new PR.

**Review by eapolinario — APPROVED** [review]

**Comment by samhita-alla** [comment]
@Ln11211, seems like the content's not rendering properly. Would you mind creating a flytekit PR to remove the `To fix this error, ...` bullet point and have `troubleshooting guide` instead of `flyte:` in `refer to the flyte:` statement in the note.

image

**Comment by Ln11211** [comment]
> @Ln11211, seems like the content's not rendering properly. Would you mind creating a flytekit PR to remove the `To fix this error, ...` bullet point and have `troubleshooting guide` instead of `flyte:` in `refer to the flyte:` statement in the note.
> 
> image

Yup. I'll do it right now

**Comment by Ln11211** [comment]
> @Ln11211, seems like the content's not rendering properly. Would you mind creating a flytekit PR to remove the `To fix this error, ...` bullet point and have `troubleshooting guide` instead of `flyte:` in `refer to the flyte:` statement in the note.
> 
> image

Here it is #1229 @samhita-alla
